content/doc/book/installing/_docker-for-tutorials.adoc

```diff
@@ -41,36 +75,69 @@ copying and pasting this annotation-free version here:
 +
 [source]
 ----
-docker run \
-  --rm \
-  -u root \
-  -p 8080:8080 \
-  -v jenkins-data:/var/jenkins_home \
-  -v /var/run/docker.sock:/var/run/docker.sock \
-  -v "$HOME":/home \
-  jenkinsci/blueocean
+docker container run --name jenkins-tutorial --rm --detach \
+  --network jenkins --env DOCKER_HOST=tcp://docker:2376 \
+  --env DOCKER_CERT_PATH=/certs/client --env DOCKER_TLS_VERIFY=1 \
+  --volume jenkins-data:/var/jenkins_home \
+  --volume jenkins-docker-certs:/certs/client:ro \
+  --volume "$HOME":/home --publish 8080:8080 jenkinsci/blueocean
 ----
 . Proceed to the <<setup-wizard,Setup wizard>>.
 
 
 ==== On Windows
 
 . Open up a command prompt window.
+. Create a link:https://docs.docker.com/network/bridge/[bridge network] in
+  Docker using the following
+  link:https://docs.docker.com/engine/reference/commandline/network_create/[`docker network create`]
+  command:
++
+[source]
+----
+docker network create jenkins
+----
+. Create the following link:https://docs.docker.com/storage/volumes/[volumes] to
+  share the Docker client TLS certificates needed to connect to the Docker
+  daemon and persist the Jenkins data using the following
+  link:https://docs.docker.com/engine/reference/commandline/volume_create/[`docker volume create`]
+  commands:
++
+[source]
+----
+docker volume create jenkins-docker-certs
+docker volume create jenkins-data
+----
+. In order to execute Docker commands inside Jenkins nodes, download and run
+  the `docker:dind` Docker image using the following
+  link:https://docs.docker.com/engine/reference/commandline/container_run/[`docker container run`]
+  command:
++
+[source]
+----
+docker container run --name jenkins-docker --rm --detach ^
+  --privileged --network jenkins --network-alias docker ^
+  --env DOCKER_TLS_CERTDIR=/certs ^
+  --volume jenkins-docker-certs:/certs/client ^
+  --volume jenkins-data:/var/jenkins_home ^
+  --volume "%HOMEDRIVE%%HOMEPATH%":/home ^
+  --publish 2376:2376 docker:dind
```

Comment:
Got "Error starting userland proxy: listen tcp 0.0.0.0:2376: bind: An attempt was made to access a socket in a way forbidden by its access permissions."

Diagnose with `netstat -ano | findstr 2376`  => no results.

Trying to know if the port is a "reserved one" using `netsh int ipv4 show excludedportrange protocol=tcp` => Range from 2280 to 2379 is reserved. It seems to be a problem with Hyper-V.

Workaround: moving to `2476:2376` or ⚠️ not publishing any port.

Reply:
Removed.